pkg/cmd/roachtest/tests/failure_injection.go

```diff
@@ -93,6 +102,11 @@ func (t *failureSmokeTest) run(
 		return err
 	}
 	l.Printf("%s: Running Restore(); details in %s.log", t.failureName, file)
+	if err = c.AddGrafanaAnnotation(ctx, l, grafana.AddAnnotationRequest{
+		Text: fmt.Sprintf("%s restored", t.testName),
```

Comment:
Nit: I probably mentioned this before... terminology-wise, `recovered` seems more unequivocal than `restored`, since the latter is usually used in the context of backup/restore.

Reply:
Sure, renamed. I also have no strong preference which is why I just went the easy route of looking at what chaos mesh used :)